includes/pg-functions.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+
+
+/**
+ * Check if logs table exists
+ */
+function isPgLogsTableCreated()
+{
+    global $wpdb;
+    $tableName = $wpdb->prefix . PG_LOGS_TABLE_NAME;
+    if ($wpdb->get_var("SHOW TABLES LIKE '$tableName'") == $tableName) {
+        return true;
+    }
+
+    return false;
+}
+
+
+
+function createPgLogsTable()
+{
+    global $wpdb;
+    $charset_collate = $wpdb->get_charset_collate();
+    $LogsTableName   = $wpdb->prefix . PG_LOGS_TABLE_NAME;
+    $sqlQuery        = "CREATE TABLE $LogsTableName ( 
+    id int NOT NULL AUTO_INCREMENT,
+    log text NOT NULL, 
+    createdAt timestamp DEFAULT CURRENT_TIMESTAMP,
+    UNIQUE KEY id (id)) 
+    $charset_collate";
+    require_once(ABSPATH . 'wp-admin/includes/upgrade.php');
+    dbDelta($sqlQuery);
+}
+
+/**
+ * @param null $exception
+ * @param null $message
+ */
+function insertLogEntry($exception = null, $message = null)
+{
+    global $wpdb;
+    if (! isPgLogsTableCreated()) {
+        createPgLogsTable();
+    }
+    $logEntry = new Pagantis\ModuleUtils\Model\Log\LogEntry();
+    if ($exception instanceof \Exception) {
+        $logEntry = $logEntry->error($exception);
+    } else {
+        $logEntry = $logEntry->info($message);
+    }
+    $tableName = $wpdb->prefix . PG_LOGS_TABLE_NAME;
+    $wpdb->insert($tableName, array('log' => $logEntry->toJson()));
+}
+
+/**
+ * @return bool
+ */
+function areDecimalSeparatorEqual()
+{
+    $decimalSeparator                = getPgSimulatorDecimalSeparatorConfig();
```

Comment:
Try not tu use unnecesary blank spaces ($var = 'val'; )

Reply:
will correct this